tests/robot-tests/tests/libs/common.robot

```diff
@@ -342,6 +345,11 @@ user clicks link
     ${element}=  get child element  ${parent}  link:${text}
     user clicks element  ${element}
 
+user clicks link when available
+    [Arguments]   ${text}  ${parent}=css:body
+    user waits until page contains link  ${text}
```

Comment:
There is already a `user waits until parent contains element` inside `user clicks link` itself, so I'm not sure this additional wait is necessary?

Reply:
OK cool will remove, thanks!